tests/x11regressions/empathy/empathy_irc.pm

```diff
@@ -44,7 +45,14 @@ sub run() {
     };
 
     # join openqa channel
-    send_key "ctrl-j";
+    if (sle_version_at_least('12-SP2')) {
+        assert_and_click 'empathy-menu';
+        send_key_until_needlematch "empathy-menu-rooms", "down";
+        assert_and_click 'empathy-menu-joinrooms';
+    }
+    else {
+        send_key "ctrl-j";
+    }
```

Comment:
As you have a bug to track this not working on sp2 I recommend you always try ctrl-j and record a soft failure and apply your workaround if the corresponding action has no effect.